extensions/amp-story/1.0/amp-story.js

```diff
@@ -2508,7 +2508,11 @@ export class AmpStory extends AMP.BaseElement {
    */
   onMutedStateUpdate_(isMuted) {
     isMuted ? this.mute_() : this.unmute_();
-    this.element.toggleAttribute(Attributes.MUTED, isMuted);
+    this.mutateElement(() => {
+      isMuted
+        ? this.element.setAttribute(Attributes.MUTED, isMuted)
+        : this.element.removeAttribute(Attributes.MUTED);
```

Comment:
nit: to clear an attribute value, around the codebase we just do `setAttribute(Attributes.MUTED, '')`. Not a required change though